sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -258,23 +257,32 @@ def body_as_json(self, encoding='UTF-8'):
         except Exception as e:
             raise TypeError("Event data is not compatible with JSON type: {}".format(e))
 
+    def encode_message(self):
+        return self.message.encode_message()
 
-class Offset(object):
+
+class BatchSendEventData(EventData):
```

Comment:
Is this an internal only class?

Reply:
yes, so I didn't put it into __init__.py

Reply:
changed class name to _BatchSendEventData